### PR TITLE
Add example systemcard for demo

### PR DESCRIPTION
**Pull request by ravimeijerrig**
# Description

In this PR we:
- Add a systemcard example usecase for the demo based on the WOZ-waarde bepaling van de gemeente Oirschot.

Considerations:
- The systemcard is in Dutch, do we want that? 
- The links now include a name and a link, this needs to be changed in the schema too. 

Link all GitHub issues fixed by this PR.
Make sure to first open an issue, get community approval and only then create Pull Request to resolve it.
All Pull Requests must have an issue attached to them

Resolves #

## Checklist

Please check all the boxes that apply to this pull request using "x":

-   [x] I have tested the changes locally and verified that they work as expected.
-   [x] I have followed the project's coding conventions and style guidelines.
-   [x] I have rebased my branch onto the latest commit of the main branch.
-   [x] I have squashed or reorganized my commits into logical units.
-   [x] I have read, understood and agree to the [Developer Certificate of Origin](../blob/main/DCO.md), which this project utilizes.

### CODE REVIEW

File: example_system_card/system_card.yaml [hunk 1]
```diff
@@ -75,4 +82,5 @@ models:
 
 assessments:
   - !include assessments/iama.yaml
+  - !include assessments/dpia.yaml
```
Comment (laurensWe) [inline example_system_card/system_card.yaml:85]: Maybe also include this one in this PR? Like the real assessment or some kind of mock, I see this to be failing something else

Comment (ravimeijerrig) [reply example_system_card/system_card.yaml:85]: We don't have a mock of the DPIA, so I'll take it out for now

**Review by laurensWe — DISMISSED** [review]
LGTM :)

**Review by laurensWe — APPROVED** [review]
Thanks for fixing the widget :) LGTM

**Pull request merged**